includes/Modules/Ads/Settings.php

```diff
@@ -58,6 +63,7 @@ protected function get_default() {
 	 *
 	 * @since 1.122.0
 	 * @since 1.126.0 Added new settings fields for PAX.
+	 * @since n.e.x.t Added new settings fields for PAX.
```

Comment:
This comment just repeats the previous one. Perhaps let's add which fields have been changed exactly?

Reply:
Update to state that the customerID and userID fields were passed.